report/renderer/performance-category-renderer.js

```diff
@@ -167,6 +167,24 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
     return 'diagnostic';
   }
 
+  /**
+   * @param {LH.ReportResult.AuditRef[]} metrics
+   */
+  _reorderMetrics(metrics) {
+    const col1 = metrics
+      .splice(0, Math.floor(metrics.length / 2))
```

Comment:
seems like this should be `Math.ceil` (although, again, I'm not entirely sure what the desired order is :)